configs/eslint-config-compass/index.js

```diff
@@ -20,6 +20,7 @@ const reactConfigurations = [
 const testConfigurations = ['plugin:mocha/recommended'];
 const testRules = {
   'mocha/no-exclusive-tests': 'error',
+  'mocha/no-hooks-for-single-case': 'off'
```

Comment:
How do ya'll feel about this eslint change? Is it nice to have on?

Reply:
IIRC Maurizio already disabled it in another PR even. Totally up for disabling it, doesn't feel like it brings any value to us

Reply:
Please make the change. I already had to disable it in e2e tests for things like the time to first query test where I didn't want the cleanup after the test to be part of the time to first query so I had to use an after() even of there's just on it(). There are also places where the describe will need multiple it in it but because I'm only doing one test per area for now it only has one. but I still want to put the setup in a before() as it will just inevitably move out.